pkg/trace/agent/agent.go

```diff
@@ -605,6 +612,48 @@ func isManualUserDrop(pt *traceutil.ProcessedTrace) bool {
 	return dm == manualSampling
 }
 
+// errorSampling reports trace chunks with errors and tags spans as Error Tracking Backend Standalone.
+// Also sets "DroppedTrace" on the chunk.
+func (a *Agent) errorSampling(now time.Time, ts *info.TagStats, pt *traceutil.ProcessedTrace) (keep bool, numEvents int) {
+	sampled := a.runErrorSampler(now, *pt)
+	numEvents = len(a.getAnalyzedEvents(pt, ts))
```

Comment:
do we really need analyzed spans when only this error sampler is enabled? 🤔

Reply:
tbh I don't know a lot about analyzed spans but as it is set to `true` when running the error sampler in `runSamplers()`, I figured I would have it as well

Reply:
Set `setAnalyzedSpans` bool to `false` in https://github.com/DataDog/datadog-agent/pull/30065/commits/96d9668e722c806ed325dc2ae8f6950e363eae39